frontend/src/app/prompt/core.cljs

```diff
@@ -269,17 +274,22 @@
                "it. You won't need to open the logs at all."))]])
 
 (defn prompt []
-  (cond
-    (= @status "error")
-    (render-error @error-title @error-description)
-
-    ;; TODO If we are already in a two-column-layout, we should print only
-    ;; a small loading icon somewhere, not a jumbotron
-    (= @status "waiting")
-    (loading-screen "Getting a response from the server")
-
-    @form
-    (two-column-layout)
-
-    :else
-    (prompt-only)))
+  (let [query-url (-> js/window .-location .-search js/URLSearchParams. (.get "url"))]
+    (if query-url
+      (do
+        (send)
+        (loading-screen "Getting a response from the server"))
```

Comment:
Did you try this with an actual https://github.com/fedora-copr/logdetective ? I haven't yet, but my suspicion is that this will show the "Getting a response from the server" line forever. The `query-url` will be defined even after we receive the data from the server, so we will go back here, instead of going to the else branch (the whole `cond` expression).

Reply:
Still haven't tried with the actual `logdetective` service, but I think I am right. The `send` function defines these two handlers:

https://github.com/fedora-copr/logdetective-website/blob/13a7827fa64eeffd75a9ce375c801e078121248d/frontend/src/app/prompt/core.cljs#L68-L82

If you put `(js/console.log "Hello")` or `(js/alert "Hello")` in them, you can see that the response handled almost immediately (in my case) but the page still shows "Getting a response from the server"